tests/components/hue/conftest.py

```diff
@@ -32,6 +30,13 @@ def no_request_delay():
         yield
 
 
+@pytest.fixture(autouse=True)
+def no_check_migration():
+    """Make sure the check_migration code is not called in regular tests."""
+    with patch("homeassistant.components.hue.check_migration"):
```

Comment:
It would be better to patch `is_v2_bridge` of the library to make the tests more robust for internal changes.

Reply:
ok, I can adjust but will be some more work as the is_v2_bridge is actually used in some tests and mocked there.

Reply:
Pytest test fixtures can be overridden per test with `pytest.mark.parametrize`.

Reply:
Or override it per module by defining a local fixture function with the same name.

Reply:
@MartinHjelmare I've adjusted the tests now